pages/docs/concepts/asyncapi-document/reply-info.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Adding reply Info
+weight: 210
+---
+
+Request/Reply is a common messaging or communication pattern where one component or service, known as the requestor, sends a request to another component or service, known as the replier and waits for a response. This pattern is often used when a component needs to initiate an action and receive a specific response in return either synchronously or asynchronously. This communication pattern is typically asynchronous, meaning that the requester does not block and wait for an immediate response. Instead, it can continue processing other tasks or even send out other requests while waiting for the reply to arrive. 
+
+In case of multiple requests made, each request is processed independently and sends the reply/response to the corresponding requestor when ready.
+
+The request/reply pattern in AsyncAPI works in the same fashion while supporting all the different sub-patterns. Irrespective of the sub-pattern you would like to implement, the Request/reply pattern can be implemented in the asyncAPI document using actions in the `Operation` object.
+
+You can add the reply info using the `Operation Reply` object under the `Operation` object. The `reply` object represents the response message.
+
+Here is diagram to illustrate the working of a basic request/reply pattern:
+```mermaid
+sequenceDiagram
+    participant Requester as Requester
+    participant Responder as Responder
+    Requester->>Responder: Send Request
+    activate Responder
+    Responder-->>Requester: Send Reply
+    deactivate Responder
+```
+
+## Sub-patterns
+
+### Dynamic response Channel
+There are situations where you do not know the reply channel at the design time. Instead, the reply channel is dynamically determined at runtime, based on factors such as the request message payload or header. 
+
+In this case, since you don't know what the address of the reply channel is yet, you can either assign null to the `address` property or omit the property entirely indicating that the address is not known yet. You can now use `Operation Reply Address` object to dynamically specify where the reply has to be sent. The `Operation Reply Address` object has a property called `location` which essentially takes in a runtime expression that specifies the address of the reply channel.
```

Comment:
```suggestion
In this case, since you don't know what the address of the reply channel is yet, you can either assign null to the `address` property or omit the property entirely indicating that the address is not known yet. You can use `Operation Reply Address` object to dynamically specify where the reply has to be sent. The `Operation Reply Address` object has a property called `location` which essentially takes in a runtime expression that specifies the address of the reply channel. 
```

let's not write `now` as it indicates previously it was not available, but in docs we should already write about current state, assume someone new reads docs, and is not interested that before it was not possible, these are release notes for.

you can also add at the end something like
"With such an expression you can specify that, for example, reply is sent to the channel specified by the requestor in the request header under specific property, like for example `REPLY_TOPIC`" <- definitely can be written better 😄  it is just a suggestion

in general about this paragraph, problem is that first sentence talks about "address" and then there is operation reply mentioned, and it is not clear that the "address" that you can omit or set to null, is part of the channel, that the operation with reply references too